### ISSUE TITLE
M3u8 cannot be cached,Can the author perfect it?

### ISSUE THREAD

**Issue by lzjin** [danikula/AndroidVideoCache#255]
M3u8 cannot be cached,Can the author perfect it?

**Comment by karanSignity** [comment]
Did you find any solutions?

**Comment by JeffMony** [comment]
https://github.com/JeffMony/MediaLocalProxyLib
This project can cache M3U8's video resources.

**Comment by Mr-Akash** [comment]
@JeffMony I don't think this cache the m3u8 videos, I have tried the above link, but I didn't get the result. I am searching for the same.

**Comment by JeffMony** [comment]
> @JeffMony I don't think this cache the m3u8 videos, I have tried the above link, but I didn't get the result. I am searching for the same.

What is the link ? Could you send it to me ? I try it.

**Comment by Mr-Akash** [comment]
@JeffMony The github link you have provided for m3u8 videos cache( https://github.com/JeffMony/MediaLocalProxyLib ).Any solution I need help with this.

**Comment by JeffMony** [comment]
> @JeffMony The github link you have provided for m3u8 videos cache( https://github.com/JeffMony/MediaLocalProxyLib ).Any solution I need help with this.

Could you describe the problems when you use the library(https://github.com/JeffMony/MediaLocalProxyLib)

I develop it, it can cache m3u8 video.

**Comment by Mr-Akash** [comment]
@JeffMony Actually I need to play video in vertical viewpager and i am using exoplayer to play m3u8 video, but I didn't find any solution to store cache for m3u8. I need to implement when video fully streamed gets played again in loop without buffering and streaming again, is it possible to do so.

**Comment by JeffMony** [comment]
> @JeffMony Actually I need to play video in vertical viewpager and i am using exoplayer to play m3u8 video, but I didn't find any solution to store cache for m3u8. I need to implement when video fully streamed gets played again in loop without buffering and streaming again, is it possible to do so.

Yes.

**Comment by hzz9600** [comment]
https://github.com/hzz9600/VideoCacheSample
短视频预加载 short-video preload
支持m3u8缓存 support m3u8 cache
